scripts/docker-build.sh

```diff
@@ -6,6 +6,8 @@ PIPELINE_PACKAGE=${PIPELINE_PACKAGE:-"general"}
 PIPELINE_FAMILY=${PIPELINE_FAMILY:-"general"}
 PIP_VERSION="${PIP_VERSION:-22.2.1}"
 DOCKER_IMAGE="${DOCKER_IMAGE:-pipeline-family-${PIPELINE_FAMILY}-dev}"
+DOCKER_PLATFORM="${DOCKER_PLATFORM:-linux/amd64}"
```

Comment:
Did something in particular trigger adding this default assignment? I don't _think_ we want a default for DOCKER_PLATFORM. If you see down on line 18-21 we conditionally add the flag only if set, this means that if the caller doesn't explicitly pass this in we let docker just do the right thing (which I believe is to build for the host architecture).

Reply:
Good catch, I will update this tomorrow